api/routers/repository.py

```
@@ -134,26 +135,26 @@ async def get_repository_by_id(
     try:
-        overview = query_service.get_repo_overview(repo_id)
-        if not overview:
-            raise HTTPException(status_code=404, detail="Repository not found")
-
         deleted = query_service.delete_repository(repo_id)
 
+        # Always clean up Firestore regardless of whether Neo4j had the repo
+        try:
+            firebase_service.delete_repo_metadata(user["uid"], username, repo_name)
+        except Exception:
+            pass  # Firestore doc may not exist; non-fatal
```

Ruff (0.15.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)